blocks/blogs/blogs.js

```diff
@@ -55,26 +55,29 @@ async function createBlogCard(blogData) {
   return arcticleDiv;
 }
 
-async function generateCardsView(block) {
+function generateCardsView(block) {
   const blogsContainer = block.querySelector('.blogs-cards-container');
-  const blogsDataArray = await callMockBlogAPI();
-  const entriesToProcess = blogsDataArray.length;
-  /**
-   * Loop through the blogsDataArray and create a blog card for each blog entry.
-   * Append the blog card to the blogsContainers column. Each column will have 2 blog cards.
-   */
-  // eslint-disable-next-line no-plusplus
-  for (let i = 0; i + 1 < entriesToProcess; i += 2) {
-    const blogsColumn = document.createElement('div');
-    blogsColumn.className = 'blogs-container-column';
-    // eslint-disable-next-line no-await-in-loop
-    const blogCard1 = await createBlogCard(blogsDataArray[i]);
-    // eslint-disable-next-line no-await-in-loop
-    const blogCard2 = await createBlogCard(blogsDataArray[i + 1]);
-    blogsColumn.appendChild(blogCard1);
-    blogsColumn.appendChild(blogCard2);
-    blogsContainer.appendChild(blogsColumn);
-  }
+  fetchData(`${getHostUrl()}/scripts/mock-blogdata.json`, async (error, blogsDataArray = []) => {
```

Comment:
if I am right, when we will have actual APIs then we just need to replace `${getHostUrl()}/scripts/mock-blogdata.json` this line with a method which will fetch the data and response in json or whatever we need in processing ?

Reply:
yes